com/williamfiset/algorithms/sorting/RadixSort.java

```diff
@@ -1,12 +1,73 @@
-/**
- * NOTE: This file is still under development
- **/
-package com.williamfiset.algorithms.sorting;
-
-public class RadixSort {
-
-  public static void radixSort(int[] ar) {
-    
-  }
-
-}
+

```

Comment:
Add package as the first line:

package com.williamfiset.algorithms.sorting;

Reply:
Ok. I'll add that.